specs/001-rss-rss/contracts/rss-api.yaml

```
@@ -0,0 +1,116 @@
+          type: string
+          format: uri
+          description: RSS feed website URL
+        lastBuildDate:
+          type: string
+          format: date-time
+          description: Last update timestamp
+        articles:
+          type: array
+          items:
+            $ref: '#/components/schemas/Article'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)